src/Routes/DefaultPage/DefaultPage.tsx

```diff
@@ -185,10 +185,20 @@ const DefaultPage = () => {
   const [perPage, setPerPage] = useState<number>(10);
   const offset = (page - 1) * perPage;
   const [isLoading, setIsLoading] = useState<boolean>(true);
+  const navigate = useNavigate();
 
   console.log('INFO:DefaultPage render');
 
   useEffect(() => {
+    if (appContext.rbac.isLoading) {
+      console.debug('rbac is loading yet');
```

Comment:
Nit: I'd remove both debug statements. They are useful in development, but they IMHO. clutter the console later when developing something else.